code/game/objects/items/weapons/twohanded.dm

```diff
@@ -670,6 +670,8 @@
 
 /obj/item/twohanded/singularityhammer/proc/vortex(turf/pull, mob/wielder)
 	for(var/atom/movable/X in orange(5, pull))
+		if(isobserver(X))
+			continue
```

Comment:
While this does work, we should check `move_resist` instead. So something like `if(X.move_resist == INFINITY)` would be better and can cover other potential cases too, where it's pulling stuff we don't want it to.

Reply:
smart, will do.